backend/src/apiserver/template/template.go

```diff
@@ -199,7 +199,7 @@ func modelToParametersMap(modelParameters string) (map[string]string, error) {
 	}
 	err := json.Unmarshal([]byte(modelParameters), &paramsMapList)
 	if err != nil {
-		return nil, err
+		return nil, util.NewInternalServerError(err, "error marshalling model parameters")
```

Comment:
Should it be `unmarshalling`?

Reply:
Thank you for finding this out! I will change it.